examples/move/closed_loop/sources/closed_loop.move

```diff
@@ -351,105 +343,121 @@ module closed_loop::closed_loop {
         vec_set::insert(&mut request.approvals, type_name::get<W>())
     }
 
+    // === Rule Config ===
+
+    /// Add a `Config` for a `Rule` in the `TokenPolicy`. Some rules may require
```

Comment:
I think an example of when this is/isn't required would be useful--having a bit of trouble getting the intuition.